### PR TITLE
Add upload limit text

### PR DESCRIPTION
**Pull request by yinlinchen**

### CODE REVIEW

File: app/views/generic_files/upload/_uploadlimit.html.erb [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<label>
+  <strong> Upload limit: Maximum upload size is limited to 200MB each file. Maximum upload total size is limited to 500MB. Maximum number of files allowed to be uploaded is 100.</strong>
```
Comment (rotated8) [inline app/views/generic_files/upload/_uploadlimit.html.erb]: I would rewrite this as "Upload limits: You may upload up to 100 files at a time. Each file cannot be larger than 200 MBs. Your upload cannot be larger than 500 MBs in total."

Comment (ghoster) [inline app/views/generic_files/upload/_uploadlimit.html.erb]: @zhiwuxie  Please give final version

Comment (yinlinchen) [inline app/views/generic_files/upload/_uploadlimit.html.erb]: @zhiwuxie  Please give final version

**Comment by yinlinchen** [comment]
Updated.

**Comment by rotated8** [comment]
:+1:

**Pull request merged**